source/class/qx/ui/form/FileSelectorButton.js

```diff
@@ -60,9 +60,19 @@
  * [MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications)
  */
 
-let InputElementIdCounter = 0;
 qx.Class.define("qx.ui.form.FileSelectorButton", {
```

Comment:
```suggestion
let InputElementIdCounter = 0;
qx.Class.define("qx.ui.form.FileSelectorButton", {
```

Reply:
Correct me if I'm wrong, but if we transpile using babel preset to ES5 (IE11) it will became a globar (var InputElementIdCounter=0)

Reply:
@evandropoa I am pretty sure it does not, since the individual files get wrapped ... but please test if you are unsure.

Reply:
Hi, you're right, the qx compiler will wrap a file declaration inside an IIFE function, I didn't know that, but after looking at the compiler source, I can see it.
Just to note that only Babel transpilling will produce a global variable, but the qx compiler makes it safer, WELL DONE.